dev/tools/dartdoc.dart

```diff
@@ -139,7 +139,6 @@ Future<void> main(List<String> arguments) async {
   final List<String> dartdocArgs = <String>[
     ...dartdocBaseArgs,
     '--allow-tools',
-    '--enable-experiment=non-nullable',
```

Comment:
Why did you need to disable this?

Reply:
See the release notes -- this flag is on by default now.